benchmark/src/main/java/io/servicefabric/transport/protocol/ProtostuffMessageSerializerBenchmark.java

```diff
@@ -0,0 +1,109 @@
+package io.servicefabric.transport.protocol;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+
+import org.openjdk.jmh.annotations.*;
+
+import com.google.common.collect.ImmutableMap;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.servicefabric.cluster.gossip.Gossip;
+import io.servicefabric.cluster.gossip.GossipRequest;
+import io.servicefabric.transport.TransportHeaders;
+
+@Fork(2)
+@State(Scope.Thread)
+@Threads(4)
+@Warmup(iterations = 5)
+@Measurement(iterations = 5)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+public class ProtostuffMessageSerializerBenchmark {
```

Comment:
@antonkharenko refactored gossip benchmark test in PR#26 (but still with headers)